cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -1418,6 +1429,21 @@ impl<T: Config> Pallet<T> {
 		CustomValidationHeadData::<T>::put(head_data);
 	}
 
+	/// Send the ump signals
+	#[cfg(feature = "experimental-ump-signals")]
```

Comment:
Wondering if it would be a good idea to hide the `select_core_for_child  runtime API behind this feature ?

Reply:
I don't think it helps us. The parachain can implement the API without any problem. I introduced the compile-time feature because otherwise the relay chain would have rejected these candidates